### ISSUE TITLE
1.0.10 Recording Importer Issue

### ISSUE THREAD

**Issue by semzor** [blindsidenetworks/scalelite#471]
Importer doesn't import recordings from spool directory. Log related to this that I could find:

``Failed to import recording: undefined method `casecmp?' for nil:NilClass``

Do I need to explicitly set `RECORDING_DISABLED=false` in env file?

**Comment by wolbernd** [comment]
Can confirm. Here is the full stacktrace:
```
W, [2021-03-01T11:08:50.490144 #6]  WARN -- : /srv/scalelite/lib/recording_importer.rb:12:in `import': undefined method `casecmp?' for nil:NilClass (NoMethodError)
        from /srv/scalelite/lib/tasks/recordings.rake:17:in `block (4 levels) in <main>'
        from /srv/scalelite/lib/tasks/recordings.rake:16:in `each'
        from /srv/scalelite/lib/tasks/recordings.rake:16:in `block (3 levels) in <main>'
        from /srv/scalelite/vendor/bundle/ruby/2.6.0/gems/listen-3.1.5/lib/listen/event/config.rb:23:in `call'
        from /srv/scalelite/vendor/bundle/ruby/2.6.0/gems/listen-3.1.5/lib/listen/event/processor.rb:115:in `_process_changes'
        from /srv/scalelite/vendor/bundle/ruby/2.6.0/gems/listen-3.1.5/lib/listen/event/processor.rb:19:in `block in loop_for'
        from /srv/scalelite/vendor/bundle/ruby/2.6.0/gems/listen-3.1.5/lib/listen/event/processor.rb:15:in `loop'
        from /srv/scalelite/vendor/bundle/ruby/2.6.0/gems/listen-3.1.5/lib/listen/event/processor.rb:15:in `loop_for'
        from /srv/scalelite/vendor/bundle/ruby/2.6.0/gems/listen-3.1.5/lib/listen/event/loop.rb:84:in `_wait_for_changes'
        from /srv/scalelite/vendor/bundle/ruby/2.6.0/gems/listen-3.1.5/lib/listen/event/loop.rb:42:in `block in setup'
        from /srv/scalelite/vendor/bundle/ruby/2.6.0/gems/listen-3.1.5/lib/listen/internals/thread_pool.rb:6:in `block in add'

**Comment by wolbernd** [comment]
Small Update: The problem doesn't occur with v1.0.9 so going back to this version (at least for the recording importer) is a workaround.

**Comment by jfederico** [comment]
@wolbernd yes, the issue is how that validation is implemented, if the variable does not exist it triggers the error. You should get it solved in 1.0.10 by adding that variable. RECORDING_DISABLED=false
